.github/workflows/python_test.yml

```diff
@@ -0,0 +1,50 @@
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+      - enable-tox
+
+name: pythontest
+
+jobs:
+  test_tox:
+    name: run-tox
+    runs-on: ubuntu-18.04
+    timeout-minutes: 25
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '11' # The JDK version to make available on the path.
+      - name: Download and Run Pravega standalone
+        run: |
+          wget https://github.com/Tristan1900/pravega/releases/download/hostname/pravega-0.9.0-2711.b899fe6-SNAPSHOT.tgz
+          tar -xzvf pravega-0.9.0-2711.b899fe6-SNAPSHOT.tgz
+          pravega-0.9.0-2711.b899fe6-SNAPSHOT/bin/pravega-standalone > pravega.log 2>&1 &
+          sleep 120 && echo "Started standalone"
```

Comment:
It would be good if we could identify a log line when it is ready and wait for that to be logged.